test/e2e/index.test.ts

```diff
@@ -301,5 +301,29 @@ describe('e2e', () => {
         ),
       ).to.equal('1/31/2025, 4:05:00 PM');
     });
+
+    // https://github.com/mui-org/material-ui-x/issues/3613
+    it('should not lose cell focus when scrolling with arrow down', async () => {
+      await renderFixture('DataGridPro/KeyboardNavigationFocus');
+
+      async function keyDown() {
+        await page.keyboard.down('ArrowDown');
+        // wait between keydown events for virtualization
+        await sleep(100);
```

Comment:
I tried to avoid this, but virtualization needs some time in order to expose the bug